homeassistant/components/homekit_controller/config_flow.py

```diff
@@ -245,7 +243,7 @@ def __init__(self):
             },
             errors=errors,
             data_schema=vol.Schema({
-                vol.Required("pairing_code"): str,
+                vol.Required('pairing_code'):  vol.All(cv.string, vol.Strip),
```

Comment:
I don't think we can use custom validators here since we depend on voluptuous-serialize for the schema validation. So don't use `cv.string`.

Reply:
Is there something else I should instead? Is str ok?

Reply:
(I've gone ahead and changed it to just vol.Strip in the meantime)

Reply:
I guess we can do:
```py
vol.All(str, vol.Strip)
```

Reply:
Ok i've switched it to that way.